src/test/java/com/akademia/projectplanner/validation/TaskValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+package com.akademia.projectplanner.validation;
+
+import com.akademia.projectplanner.dto.TaskDto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskValidatorTest {
+
+  private TaskDto taskDto;
+
+  @BeforeEach
+  void beforeEach() {
+    taskDto = new TaskDto();
+  }
+
+  @Test
+  void testTaskNonBlank() {
+    taskDto.setStatus("In progress");
+    taskDto.setName("Task number 1");
+    assertFalse(TaskValidator.hasBlankNameOrStatus(taskDto));
+  }
+
+  @Test
+  void testTaskBlankNameAndStatus() {
+    taskDto.setStatus("");
```

Comment:
jeszcze bym rozważył spację, co prawda używamy isBlank() ale jak ktoś w przyszłości zmieni to nie mamy takiego testu, dodaj w tym samym teście drugi taki sam obiekt tylko ze spacjami i zrób wiele asercji (może być assertAll)
- tylko nie nazywamy wtedy poprzez numerki tylko lepiej taskWithWhitespace / taskWithEmptyString lub coś w ten deseń

P.S jesli gdzieś jeszcze widzisz, że Twoim zdaniem można zamiast dwóch testów zrobić jeden ale z wieloma obiektami i asercjami to śmiało